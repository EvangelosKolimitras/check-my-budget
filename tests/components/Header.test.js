import React from "react"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"
import Header from "../../src/scripts/js/Router/HeaderRouter"

test("should render Header component correctly", () => {
    const wrapper = shallow(<Header />)
    expect(toJson(wrapper)).toMatchSnapshot()
})
