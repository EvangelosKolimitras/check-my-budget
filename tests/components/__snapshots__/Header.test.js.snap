// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component correctly 1`] = `
<header>
  <h1>
    Header Section
  </h1>
  <p>
    Some to play
  </p>
  <NavLink
    className="nav-item"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    className="nav-item"
    to="/dashboard"
  >
    Dashboard
  </NavLink>
  <NavLink
    className="nav-item"
    to="/add"
  >
    Add
  </NavLink>
</header>
`;
